{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/auth/api.py

```diff
@@ -36,3 +39,49 @@ def register(self, request):
         user = create_user_account(**serializer.validated_data)
         data = serializers.AuthUserSerializer(user).data
         return response.Created(data)
+
+    @list_route(['POST', ])
+    def logout(self, request):
+        """
+        Calls Django logout method and delete the Token object
+        assigned to the current User object.
+        """
+        logout(request)
+        return response.Ok({"success": "Successfully logged out."})
+
+
+class CurrentUserView(generics.GenericAPIView):
```

Comment:
This way of implementing the resource is not really extensible, I would rather like to have it as a viewset, if possible. I'll update if I have a better way to implement this view/resource.